ui/app/AppLayouts/Wallet/controls/AccountHeaderGradient.qml

```diff
@@ -1,74 +1,77 @@
-import QtQuick 2.13
+import QtQuick 2.15
+import QtQuick.Controls 2.15
 import QtGraphicalEffects 1.15
 
 import StatusQ.Core.Theme 0.1
 
 import utils 1.0
 
-Loader {
+Control {
```

Comment:
here I've just placed the Loader in a `Control`. No other changes

The Loader `implicitHeight` and `implicitWidth` seems to have a special purpose. What's different about it it's that it's readonly. We can set the `width` and `height` on the loader, but the LayoutItemProxy can't work with `width` and `height`